src/components/bf-input-string.vue

```diff
@@ -2,24 +2,39 @@
   <bf-input-wrapper
     :label="label"
     :class="{ error: showError }"
+    :name="name"
   >
     <input
       ref="input"
       :type="type"
+      :id="name"
       :name="name"
       :value="value"
       @input="onInput"
       @blur="setTouched"
       @invalid="setTouched"
       :placeholder="placeholder"
+      :autocomplete="autocomplete"
+      :autofocus="autofocus"
+      :form="form"
+      :list="list"
+      :maxlength="maxlength"
+      :min="min"
+      :pattern="pattern"
       :disabled="disabled"
       :readonly="readonly"
+      :aria-label="name"
+      :aria-describedby="`help-text-${name}`"
     />
     <template #help-error v-if="showError">
-      {{ injectedError || error }}
+      <div role="alert">
+        {{ injectedError || error }}
+      </div>
     </template>
     <template #help-text>
-      <slot name="help-text" />
+      <div :id="`help-text-${name}`">

```

Comment:
name is not unique enough to use as an identifier

Reply:
We can generate a unique id (autoinc) from a helper function to make sure things are unique